### DISCUSSION TITLE
Not seeing runtime logs

### DISCUSSION THREAD

**Discussion by tonymullen** [vercel/community#3532] [Help]
### Summary

I am deploying a Node.js app. It appears to be running, although some data is not populating as expected. I would like to troubleshoot with server-side console.logs, but I am not seeing anything in the runtime logs for the app. Is there something in particular I need to do to activate runtime logging?

I saw https://github.com/orgs/vercel/discussions/3463 but I was not able to get logs to display with this solution. 

### Steps to Reproduce

I deploy by committing my code and running "redeploy". There are no errors and the application appears to run, but nothing appears in the runtime logs, where I expect at least some console.log messages.

**Comment by sha237gh** [comment]
I have the same exact problem. Turning into an edge function shouldn't really be the solution here and it isn't for us, as we are using the path and fs modules, we expect this to work straight out of the box. Whoever thought it was a good idea to hide those logs needs to reconsider what they are doing.

Only showing thrown errors is a very bad design. Thrown errors stop execution and it is impossible to properly handle them if console.logs aren't being propagated.

**Comment by Jooliashi** [comment]
Hello!

Can you perhaps show your function code and directory setup?
I have tried a simple example on my side with server side 
```
export default function getServerSidePropsPage() {
  return <div>YES!</div>;
}

export async function getServerSideProps(context) {
  console.log("Inside getServerSideProps");

  return {
    props: {}, 
  };
};
```

and i am able to see the console log in runtimes logs here so i'm wondering if it's your specific set up
![CleanShot 2024-02-19 at 18 22 29@2x](https://github.com/vercel/community/assets/49071013/11600216-52dc-403a-8ce4-6311ca3a2976)
